### ISSUE TITLE
Inventory sync fails with IOError: Source does not exist on first run after update docker images

### ISSUE THREAD

**Issue by jpmens** [ansible/awx#503]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY
After updating the docker images (awx-task and awx-web) and waiting a few minutes I launch the first job which requires an inventory sync. That inventory sync fails with `IOError: Source does not exist: /var/lib/awx/projects/_6__p1/hosts`

##### ENVIRONMENT

* AWX version: 1.0.1.93
* AWX install method: docker
* Ansible version:  2.4.0.0
* Operating System: CentOS 7.3
* Web Browser: Chrome

##### STEPS TO REPRODUCE

1. Update docker images

```
docker stop <awx_web and task>
docker pull ansible/awx_task
docker pull ansible/awx_web
```

2. Launch containers

```
ansible-playbook -i inventory installer.yml
```

3. Wait a few moments then trigger job template via API

##### EXPECTED RESULTS

I expected job to run

##### ADDITIONAL INFORMATION

Job fails with the following output

```
2017-10-25 15:09:01,086 INFO     awx.main.commands.inventory_import Updating inventory 3: j1
Traceback (most recent call last):
  File "/usr/bin/awx-manage", line 9, in <module>
    load_entry_point('awx==1.0.1.93', 'console_scripts', 'awx-manage')()
  File "/usr/lib/python2.7/site-packages/awx/__init__.py", line 107, in manage
    execute_from_command_line(sys.argv)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/management/base.py", line 661, in handle
    return self.handle_noargs(**options)
  File "/usr/lib/python2.7/site-packages/awx/main/management/commands/inventory_import.py", line 1000, in handle_noargs
    self.is_custom)
  File "/usr/lib/python2.7/site-packages/awx/main/management/commands/inventory_import.py", line 234, in load_inventory_source
    raise IOError('Source does not exist: %s' % source)
IOError: Source does not exist: /var/lib/awx/projects/_6__p1/hosts
```

The second launch of this same job succeeds.

**Comment by jpmens** [comment]
This same error occurred a few days ago when I upgraded to `1.0.1.77`. (I do not recall _from_ which version.)

**Comment by AlanCoding** [comment]
I'm using the docker containers, an I re-traced using these steps:

 - Created an inventory with SCM inventory source set to update on launch
 - Use this in a JT, launch it normally as sanity check
 - in container `rm -rf /projects/`
 - launch again

When I do this, I find that an inventory update is triggered. Both the JT and the inventory update employ a project sync, and the first project sync involved recreates the project folder.

This might be something more complex with the task-specific container in the installer.

**Comment by AlanCoding** [comment]
Never mind, I had the source type wrong. I am able to replicate this now.

**Comment by matburt** [comment]
Fixed in #528
